package.json

```diff
@@ -47,6 +48,7 @@
     "express-validator": "^6.2.0",
     "helmet": "^3.21.2",
     "i18n": "^0.8.4",
+    "json-comb": "^0.2.31",
```

Comment:
Question:  `jsoncomb` won't fail with a non-zero exit code if the files don't match, and the files normalized during the pre-commit hook won't get committed from this run. Is there a benefit to running here that I'm missing? (If not, this might just increase the time it takes to push without gain.)

Reply:
It doesn't look like it's running during the pre-commit hook, I think you got confused with the shrunk page.

Reply:
Ah! Sorry. The way GitHub collapsed the view made it look like it was part of husky. My bad.